app/main/views/services.py

```diff
@@ -5,41 +5,52 @@
 from ...main import main, content_loader
 from ..helpers import login_required
 from ..helpers.services import is_service_associated_with_supplier, get_signed_document_url, count_unanswered_questions
-from ..helpers.frameworks import get_framework_and_lot, get_declaration_status, get_supplier_framework_info
+from ..helpers.frameworks import (
+    get_framework_and_lot,
+    get_declaration_status,
+    get_supplier_framework_info,
+    get_framework,
+)
 
 from dmcontent.content_loader import ContentNotFoundError
 from dmapiclient import HTTPError
 from dmutils import s3
 from dmutils.documents import upload_service_documents
 
 
-@main.route('/services')
+@main.route("/frameworks/<string:framework_slug>/services")
 @login_required
-def list_services():
-    suppliers_services = sorted(
-        data_api_client.find_services(supplier_id=current_user.supplier_id)["services"],
-        key=lambda service: service['frameworkSlug'],
-        reverse=True
-    )
+def list_services(framework_slug):
+    framework = get_framework(data_api_client, framework_slug, allowed_statuses=['live'])
+
+    suppliers_services = data_api_client.find_services(
+        supplier_id=current_user.supplier_id,
+        framework=framework_slug,
+    )["services"]
 
     return render_template(
         "services/list_services.html",
-        services=suppliers_services), 200
+        services=suppliers_services,
+        framework=framework,
+    ), 200
 
 
 #  #######################  EDITING LIVE SERVICES #############################
 
 
-@main.route('/services/<string:service_id>', methods=['GET'])
+@main.route("/frameworks/<string:framework_slug>/services/<string:service_id>", methods=['GET'])
 @login_required
-def edit_service(service_id):
+def edit_service(framework_slug, service_id):
     service = data_api_client.get_service(service_id)
     service_unavailability_information = service.get('serviceMadeUnavailableAuditEvent')
     service = service.get('services')
 
     if not is_service_associated_with_supplier(service):
         abort(404)
 
+    if service["frameworkSlug"] != framework_slug:
```

Comment:
The answer is: probably yes, but we can't do it for this story and remain within time constraints.